src/test/unit_tests/test_nsfs_access.js

```diff
@@ -66,29 +75,22 @@ mocha.describe('new tests check', async function() {
         }
     });
     mocha.it('NON ROOT 1 mkdir - failure', async function() {
+        const new_folder = "folder1";
+        const new_path = path.join(full_path_root, new_folder);
         try {
-            const mkdir_res = await nb_native().fs.mkdir(NON_ROOT1_FS_CONFIG, full_path_non_root, 0o770);
+            const mkdir_res = await nb_native().fs.mkdir(NON_ROOT1_FS_CONFIG, new_path, 0o770);
             assert.fail(`non root has access to mkdir under root dir ${mkdir_res}`);
         } catch (err) {
             assert.equal(err.code, 'EACCES');
         }
     });
     mocha.it('ROOT readdir - dir created by non dir - success', async function() {
-        try {
-            const root_entries = await nb_native().fs.readdir(ROOT_FS_CONFIG, full_path_non_root);
-            assert.fail(`root has access to a folder that should not exist - ${root_entries}`);
-        } catch (err) {
-            assert.equal(err.code, 'ENOENT');
-        }
+            await nb_native().fs.readdir(ROOT_FS_CONFIG, full_path_non_root);
```

Comment:
why did you change this test?

Reply:
it was wrong. by test name it was supposed to test that root is able to access folder that was created by non-root user. and it should succeed. in practice it tested that root couldn't access a folder that doesn't exist